warehouse/migrations/versions/eb736cb3236d_drop_macaroon_description_limit.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Drop Macaroon description limit
+
+Revision ID: eb736cb3236d
+Revises: cc06bd67a61b
+Create Date: 2023-03-07 21:29:53.314390
+"""
+
+import sqlalchemy as sa
+
+from alembic import op
+
+revision = "eb736cb3236d"
+down_revision = "cc06bd67a61b"
+
+
+def upgrade():
+    op.alter_column(
+        "macaroons",
+        "description",
+        existing_type=sa.VARCHAR(length=100),
+        type_=sa.String(),
+        existing_nullable=False,
+    )
+
+
+def downgrade():
+    op.alter_column(
+        "macaroons",
+        "description",
+        existing_type=sa.String(),
+        type_=sa.VARCHAR(length=100),
+        existing_nullable=False,
+    )
```

Comment:
NB: This downgrade probably isn't sound, unless it does some silent truncation. Maybe time for another Order 227?

Reply:
Yep, I think so.